File: src/s100-tpm/s100-tpm/FormInicio.vb
Imports System.Data.SQLite



Public Class FormInicio

    Private Sub ButtonEtiquetajePromocional_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles ButtonEtiquetajePromocional.Click
        FormEtiquetajePromocional.Show()
    End Sub

    Private Sub ButtonEtiquetajeRevenue_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles ButtonEtiquetajeRevenue.Click
        FormEtiquetajeRevenue.Show()
    End Sub

    Private Sub ButtonRecogida_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles ButtonRecogida.Click
        FormRecogida.Show()
    End Sub


    Private Sub ButtonSalir_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles ButtonSalir.Click
        Me.Close()
        System.Windows.Forms.Application.Exit()
    End Sub

    Private Sub FormInicio_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        MyGlobal.sqlConnMaestros_a = New SQLiteConnection()
        MyGlobal.sqlConnMaestros_a.ConnectionString = "Data Source=tpm_maestros.db;Version=3;"
        Try
            MyGlobal.sqlConnMaestros_a.Open()
        Catch myException As SQLiteException
            MessageBox.Show("Message: " + myException.Message + "\n")
        End Try

        MyGlobal.sqlConnTransacciones_a = New SQLiteConnection()
        MyGlobal.sqlConnTransacciones_a.ConnectionString = "Data Source=tpm_transacciones.db;Version=3;"
        Try
            MyGlobal.sqlConnTransacciones_a.Open()
        Catch myException As SQLiteException
            MessageBox.Show("Message: " + myException.Message + "\n")
        End Try


        StatusBar1.Text = "Open DB OK"


    End Sub



    Private Sub ButtonSincroniza_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles ButtonSincroniza.Click
        FormSincronizacion.Show()
    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        FormInventario.Show()
    End Sub
End Class

File: src/s100-disvet/s100-disvet/FormInicio.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace s100_disvet
{
    public partial class FormInicio : Form
    {
        FormVenta myFormVenta_a;
        FormSincronizacion myFormSincronizacion_a;

        public FormInicio()
        {
            InitializeComponent();
            myFormVenta_a = new FormVenta();
            myFormSincronizacion_a = new FormSincronizacion();

        }

        private void menuItemVentas_Click(object sender, EventArgs e)
        {
            myFormVenta_a.Show();       
        }

        private void menuItemTransferir_Click(object sender, EventArgs e)
        {
            myFormSincronizacion_a.Show(); 
        }

        private void menuItemSalir_Click(object sender, EventArgs e)
        {
            System.Windows.Forms.Application.Exit();
        }


    }
}